engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -2467,6 +2473,12 @@ static uintptr_t GetExtProcAddress(const char* name, const char* extension_name,
 
         glTexParameteri(type, GL_TEXTURE_WRAP_T, GetOpenGLTextureWrap(vwrap));
         CHECK_GL_ERROR
+
+        if (g_Context->m_AnisotropySupport)
+        {
+            glTexParameterf(type, GL_TEXTURE_MAX_ANISOTROPY_EXT, dmMath::Min(max_anisotropy, g_Context->m_MaxAnisotropy));
+            CHECK_GL_ERROR
+        }
```

Comment:
Maybe check if value passed in is > 1.0, since according to the docs (https://www.khronos.org/opengl/wiki/Sampler_Object#Anisotropic_filtering):
"Any value greater than 1.0f counts as a use of anisotropic filtering."
The drive should probably not do anything in that case, but we might aswell skip trying to set the parameter in any case

Reply:
Sounds reasonable.
But >1 leaves out the case when you wish to call with 1.0f?

Reply:
I don't think an anisotropic value of 1.0 is supposed to do anything, the docs say greater than 1.0 🤔